meetup.py

```diff
@@ -0,0 +1,74 @@
+"""Meetup API code and datastore model classes.
+"""
+import logging
+
+from granary import meetup as gr_meetup
+from oauth_dropins import meetup as oauth_meetup
+from oauth_dropins.webutil.util import json_dumps, json_loads
+import webapp2
+
+from models import Source
+import util
+
+# We don't support listen
+LISTEN_SCOPES = []
+PUBLISH_SCOPES = [
+  'rsvp',
+]
+
+
+class Meetup(Source):
+  GR_CLASS = gr_meetup.Meetup
+  OAUTH_START_HANDLER = oauth_meetup.StartHandler
+  SHORT_NAME = 'meetup'
+  BACKFEED_REQUIRES_SYNDICATION_LINK = True
+  CAN_PUBLISH = True
+  URL_CANONICALIZER = util.UrlCanonicalizer(
+    domain=GR_CLASS.DOMAIN,
+    headers=util.REQUEST_HEADERS)
+
+  @classmethod
+  def button_html(cls, feature, **kwargs):
+    return super(cls, cls).button_html(feature, start_location='/meetup/publish/start', **kwargs)
```

Comment:
hmm, interesting. this gets applied to the sign-up button on the front page too though, right? which seems wrong?

Reply:
Oh that's the point I was going for - we don't support backfeed so I wanted the UX to only look at publish, otherwise it would be confusing

Reply:
right, but this is for the signup button, which should always start the signup flow, ie `/meetup/start`. i think you've hidden the listen (backfeed) UX with your new conditionals in `social_user.html`, right?

Reply:
True. I believe the issue is then that if following `/meetup/start`, you'll be logged in with your user - all good so far.

But then when attempting to publish, it requires you to go through `/meetup/publish/start`. 

If you're happy with that, then OK - just wasn't sure UX wise it was best (and it then complicates `redirect_uri`)

Reply:
right, that's expected. all the other bridgy silos work that way too. if you're only allowed one redirect url, that does complicate it, but you can write a multiplexer if you want, like the blogger one i mentioned.

and this should all be invisible plumbing and not affect the UX, right?

(i'm in chat now btw, feel free to jump in!)

Reply:
Okey dokey, I'll try it that way - won't be until tomorrow evening so let me know if there are any other review comments to address at the same time!